packages/contracts-bedrock/test/L2/L1Block.t.sol

```diff
@@ -315,3 +334,28 @@ contract L1Block_SetL1BlockValuesIsthmus_Test is L1Block_TestInit {
         assertEq(data, expReturn);
     }
 }
+
+/// @title L1Block_Uncategorized_Test
+/// @notice Tests the `L1Block` contract with a custom gas token.
+contract L1Block_Uncategorized_Test is CommonTest {
```

Comment:
Why uncategorized

Reply:
We could called like `L1Block_IsCustomGasToken_Test`, another for token name and the last one for token symbol in 3 different tests